expression_parser_test.go

```diff
@@ -218,29 +218,46 @@ func TestValidBooleanExpressionSyntax(t *testing.T) {
 }
 
 // The URLs below are not valid ODATA syntax, the parser should return an error.
-func TestInvalidExpressionSyntax(t *testing.T) {
+func TestInvalidBooleanExpressionSyntax(t *testing.T) {
 	queries := []string{
-		"()", // It's not a boolean expression
 		"(TRUE)",
```

Comment:
Add explanation comment for this line and next. I think it's similar to lines 228,233